ci-operator/step-registry/windows/e2e/operator/test/aws-disconnected/windows-e2e-operator-test-aws-disconnected-workflow.yaml

```diff
@@ -0,0 +1,10 @@
+workflow:
+  as: windows-e2e-operator-test-aws-disconnected
+  steps:
+    pre:
+    - chain: windows-e2e-operator-test-aws-disconnected-pre
+    post:
+    - chain: windows-e2e-operator-test-aws-disconnected-post
```

Comment:
could we re-use [cucushift-installer-rehearse-vsphere-ipi-disconnected-deprovision](https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/disconnected/deprovision/cucushift-installer-rehearse-vsphere-ipi-disconnected-deprovision-chain.yaml)?